src/components/results/feedback/modal/index.js

```diff
@@ -0,0 +1,97 @@
+import {faTimes} from "@fortawesome/free-solid-svg-icons";
+import {feedbackModalShowState} from "lib/recoil";
+import {useSetRecoilState} from "recoil";
+import useTranslate from "lib/hooks/use-translate";
+import Icon from "components/common/icon";
+import useLoadedValue from "lib/hooks/use-loaded-value";
+import style from "./style.scss";
+import {feedbackSurveyQuery} from "../../../../lib/recoil/feedback";
+
+export default function FeedbackModal() {
+  const setShow = useSetRecoilState(feedbackModalShowState);
+  const feedbackSurvey = useLoadedValue(feedbackSurveyQuery);
+  const translate = useTranslate();
+
+  console.log("feedbackSurvey :>> ", feedbackSurvey);
+  if(!feedbackSurvey) { return null; }
+
+  const onCancel = () => {
+    setShow(false);
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    const response = new Map(new FormData(event.target).entries());
```

Comment:
I like the idea of this, but we usually use controlled inputs and state for form data